src/Typist.jsx

```diff
@@ -8,6 +8,7 @@ export default class Typist extends Component {
 
   static propTypes = {
     children: PropTypes.node,
+    component: PropTypes.oneOfType([PropTypes.element, PropTypes.string]),
```

Comment:
this should be `oneOfType([PropTypes.func, PropTypes.string])`, if we actually want to allow component classes.
However, I'm thinking that for simplicity's sake, we should just take a string without having to allow component classes.